Scripts/Scratchpad/Extensions/lib/AV8BNA.lua

```diff
@@ -0,0 +1,55 @@
+ft = {}
```

Comment:
I've only checked #60 for now, but have an early question. Are the files in the `lib` directory written by you, or sourced from somewhere else? If latter, is there a license for those files that we should put next to them?

Reply:
Everything in this PR is original code. The lib\ files are mostly API calls that are created in this PR (tt(), etc). A few of the function calls are DCS Exports API. The reason I put them into a different directory than Disabled\ or Extensions\ is that they are not scratchpad compatible lua files. They rely on APIs from aeronautes-pit.lua. Originally they were in the Scratchpad\ pages directory but that got cluttered quickly. If you have a suggestion for other directory or names, I'd be open to discussing.

Reply:
Thanks for confirming. Location is fine with me – might make sense to group them into `lib/aeronautes-pit/*`, but I do not have a strong opinion about that.

Reply:
renamed directory to aeronautes-lib